addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22541,3 +22541,33 @@ msgstr ""
 msgctxt "#39153"
 msgid "Windowing system:"
 msgstr ""
+
+#. Description of setting with label #39156 "Override ASS / SSA subtitles styles"
```

Comment:
Great description! Could the strings below use descriptions too?

Reply:
if you mean this, i can add helper string for translators on all enum values below